app.js

```diff
@@ -19,25 +20,62 @@ var bot = new Slackbot(require('@slack/client'), env);
 var tweeter = TwitterPoster(env);
 var log = helpers.logger('App', env.isDev);
 
-function postText (message) {
-  console.log(message);
-  var text = message.item.message && message.item.message.text;
+var itemCache = LRU({
```

Comment:
It seems like `itemCache` is naturally a member of `tweetCache` and does not need to be in the gobal scope. Is this a js style thing where you don't want to define a class and you don't want to use wacky `(function() { })()` notation?

Reply:
Yeah, it's super dirty because it was the last thing I wrote and I was starting to feel sheepish about doing this instead of work. You're exactly right; the public API is `tweetCache` and I oughta break it off into a class.